rel_speed_thresh: 1.0 # threshold for filtering out moving targets based on relative speed
steering_ratio: 18.61
wheel_radius: 0.31242 # 24.6in diameter tire 
